spec/jobs/send_email_for_draft_job_applications_job_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "rails_helper"
+
```

Comment:
[Minor] Could include a test that asserts that the email is not sent when vacancy has 11+ days before expiration